fix: Inline how-to-guide (#1517)
appsmithorg/appsmith-docs: website/docs/reference/widgets/table/inline-editing.md

# Setup Table Inline Editing

This page shows you how to add and edit Table data directly from the UI through inline editing.

<VideoEmbed host="youtube" videoId="eIecDfvSOsU" title="" caption=""/>

## Prerequisites

> * A [Table widget](/reference/widgets/table) connected to a query that holds the data you want to edit and update.

Reviewer: ```suggestion
* A [Table widget](/reference/widgets/table) connected to a query with the data you want to edit and update.
```

:::

## Enable inline editing 

Enable inline editing by checking the **Editable** checkbox for the desired columns in the Table widget properties panel. Once inline editing is enabled, you can edit cell contents by clicking on the ✏️ icon or double-clicking on a cell.

<figure>
  <img src="/img/inline-22.png" style= {{width:"600px", height:"auto"}} alt="Display images on table row selection"/>
  <figcaption align = "center"><i>Enable inline editing</i></figcaption>
</figure>

## Update single row

To update a single row, follow these steps:

1). Select **Single row**  from the **Update mode** property. This enables the **Save/Discard** column with Save and Discard buttons.

2). Create a new query that uses the `updatedRow` reference property to update the relevant data.

<dd>

_Example_: Suppose you want to modify the `name`  and `phone` field in a `users` database.

```sql
UPDATE users SET 
  name = {{Table1.updatedRow.name}},
  phone = {{Table1.updatedRow.phone}}
  WHERE id = {{ Table1.updatedRow.id }};
```

</dd>

3). Click on the gear icon ⚙️ next to the **Save/Discard** column in the property pane.

> 4). Set the **onSave** event to run the update query, and **onSuccess** callback to trigger a fetch query that refreshes the data.

Reviewer: ```suggestion
4). Set the **onSave** event to run the update query, and the  **onSuccess** callback to trigger the fetch query that refreshes the table data with the updated information.
```

Reviewer: repeat this wherever the same statement is being used

Author: done

> As an alternative you can configure the **onSubmit** event in each [column's settings](/reference/widgets/table/column-settings) to run a query that saves the new data. The **onSubmit** event takes place whenever the user clicks away from the edited cell, or presses the Enter key within it. 

Reviewer: ```suggestion
Alternatively, you can configure the **onSubmit** event for each [Column](/reference/widgets/table/column-settings) to run a query that saves the new data. The **onSubmit** event is triggered when the user clicks away from the edited cell or presses the Enter key within it. 
```
